### ISSUE TITLE
IE11 Object.entries

### ISSUE THREAD

**Issue by JamesIves** [gaearon/react-hot-loader#1303]
### Description

What you are reporting:

Having an issue with IE11. I'm getting an error thrown which is preventing the site from loading. I've included the correct polyfills for IE and have react-hot-loader in my babel file.

```
SCRIPT438: Object doesn't support property or method 'entries'
react-hot-loader.development.js (1188, 1)
```

### Expected behavior

What you think should happen:

It shouldn't error!

### Actual behavior

What actually happens:

It's erroring

### Environment

React Hot Loader version:

`version "4.12.6"`

Run these commands in the project folder and fill in their results:

1.  `node -v`: `10.16.0`
2.  `npm -v`: `6.9.0`

Then, specify:

1.  Operating system: Mac OSX
2.  Browser and version: Internet Explorer 11

### Reproducible Demo

Unfortunately I don't have one.

**Comment by JamesIves** [comment]
Nevermind! This is unrelated.

**Comment by ArnaudWeyts** [comment]
@JamesIves Do you know what ended up being the cause of this? I'm running into the same issue and it seems like it is related to `react-hot-loader` 🤔 

Can't upgrade from `4.12.3` for this reason

**Comment by steven-qi** [comment]
just see this. i actually ran into very same problem. it seems the code run beofore polyfill is loaded, which cause error on ie 11. I actually opened a PR https://github.com/gaearon/react-hot-loader/issues/1304 to fix it.

**Comment by theKashey** [comment]
Fix has been released in 4.12.8, thank you for creating the PR.
